NIPTool/server/views.py

```diff
@@ -342,3 +354,34 @@ def update():
                 )
 
     return redirect(request.referrer)
+
+
+@server_bp.route("/load", methods=["POST"])
+def load():
+    """Function to load data into the database with rest"""
+
+    request_data = request.form
+    v = Validator(batch_load_schema)
+    if not v.validate(request_data):
+        return "Incomplete batch load request", 400
+
+    batch_data = parse_batch_file(request_data['result_file'])
+    load_batch(current_app.adapter, batch_data[0], request_data)
+    load_samples(current_app.adapter, batch_data, request_data['project_name'])
+    load_concentrations(current_app.adapter, request_data["concentrations"])
+
+    return redirect(request.referrer)
```

Comment:
Which page is the request referrer?

Reply:
Oh, I just didnt know what to set as return.  Do you have a suggestion?

Reply:
Return a json object with a message with 200 status if the command worked or a json with the specific error message and an error status if it fails?